testdata/generated/categories-prerelease-error.json

```diff
@@ -0,0 +1 @@
+invalid 'prerelease' query param: 'foo'
```

Comment:
nit: not a JSON file, right?

Reply:
Non-json files renamed in https://github.com/elastic/package-registry/pull/785/commits/b43b38aa2d6ed6bad66825b0a88d5dd967ced1ec.